### PR TITLE
PR for Desktop version portfolio

### PR DESCRIPTION
**Pull request by Aidan-bn**
# This PR is for portfolio Desktop version.

###  It includes the following.

- The use of media queries to make sure that portfolio project website displays correctly on different screens screens

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,839 @@
...
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-columns: (minmax(280px, 370px)) [auto-fit];
+  grid-template-columns: repeat(auto-fit, minmax(280px, 370px));
+  grid-gap: 25px;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
```
Comment (mirouhml) [inline css/styles.css:313]: - [ ] The projects grid should be set to show only 3 projects per row, as per the design of the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468), kindly fix that to match the design. 😁

image

Note: please also be mindful of the horizontal paddings and margins of this section. 😉

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,839 @@
...
+  display: flex;
+  padding: 80px 0;
+  gap: 24px;
+}
+
+@media (min-width: 768px) {
+  .form {
+    display: flex;
+    justify-content: space-between;
+  }
```
Comment (mirouhml) [inline css/styles.css:611]: - [ ] Please also fix the horizontal paddings and margins of the `form` section, as it is sticking to the page borders, and it also should match the design of the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

Right now it's like this:
image

This is the design of the Figma template:

image

File: index.html [hunk 3]
```diff
@@ -4,288 +4,286 @@
...
+               <img src="./images/bottomleft.svg" alt="imgleft" class="imgleft">
+               <img src="./images/bottomiddl.svg" alt="imgmiddle" class="imgmiddle">
+               <img src="./images/bottomright.svg" alt="imgright" class="imgright">
+           </figure>
+       </div>
+       <hr class="bottom-line">
+       <div class="medias">
+           <ul class="ul-socialmedia">
+               <li class="socialists"><a href="https://twitter.com/aidan-bn"><img src="./images/twitter.svg" alt="twitter"></a></li>
+               <li class="socialists"><a href="https://twitter.com/aidan-bn"><img src="./images/linkedin.svg" alt="linkedin"></a></li>
```
Comment (mirouhml) [inline index.html:275]: - [ ] Same thing as before for the name of the LinkedIn file, kindly fix this so it locates the right file. 😁

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,839 @@
...
+  height: 48px;
+  gap: 10px;
+  background: #36b37f;
+  color: #fff;
+  border-radius: 8px;
+  border: 0;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+}
```
Comment (mirouhml) [inline css/styles.css:729]: - [ ] Please add styling for the other states of the buttons (`active` and `hover`) as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design: 😁

image

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @Aidan-bn, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,839 @@
...
+.down {
+  width: 14px;
+  height: 8px;
+}
+
+.work {
+  display: flex;
+  flex-direction: column;
+  padding: 80px, 0, 80px, 0;
+  gap: 48px;
```
Comment (mirouhml) [inline css/styles.css:287]: - [ ] There shouldn't be any commas here, it should be like this:  `padding: 80px 0 80px 0;` kindly fix it. 😁

File: index.html [hunk 6]
```diff
@@ -4,288 +4,286 @@
...
+                   <span>Hey there. I’m Aidan</span>
+                   <span>I’m a software developer</span>
+               </h1>
+               <p>
+                   I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                   don’t hestiate to contact me.
+               </p>
+               <ul class="social">
+                   <li><a href=""><img src="./images/twitter.svg" alt=""></a></li>
+                   <li><a href=""><img src="./images/linkedin.svg" alt=""></a></li>
```
[the remark this answers is not in the document]
Comment (Aidan-bn) [reply index.html:40]: Resolved

File: css/styles.css (hunk 2 above)
Comment (Aidan-bn) [reply css/styles.css:611]: Resolved

File: index.html (hunk 3 above)
Comment (Aidan-bn) [reply index.html:275]: Resolved

File: css/styles.css (hunk 4 above)
Comment (Aidan-bn) [reply css/styles.css:729]: Resolved

File: css/styles.css (hunk 5 above)
Comment (Aidan-bn) [reply css/styles.css:287]: Resolved

**Review by algerina — APPROVED** [review]
Hi @Aidan-bn , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE

 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**